include/openPMD/Series.hpp

```diff
@@ -50,20 +50,12 @@ class Series : public Attributable
 
 public:
 #if openPMD_HAVE_MPI
-    static Series create(std::string const& filepath,
-                         MPI_Comm comm,
-                         AccessType at = AccessType::CREATE);
-#endif
-    static Series create(std::string const& filepath,
-                         AccessType at = AccessType::CREATE);
-
-#if openPMD_HAVE_MPI
-    static Series read(std::string const& filepath,
-                       MPI_Comm comm,
-                       AccessType at = AccessType::READ_ONLY);
+    Series(std::string const& filepath,
+           AccessType at,
```

Comment:
did you intentionally not put the `AccessType` last? Having it at the last position will allow us to put a good default there later on

Reply:
The idea of a default ```AccessType``` was intuitive when the factories were in place, but IMO there is no clear, sane default for the constructor now.
We could think about assuming ```READ_ONLY``` if a file exists on disk and ```CREATE``` if it doesn't. Personally, I'd prefer to explicitly state it explicitly in any case. Users would have to look up the assumed behaviour and not see it. Also, the constructor will generally not be called often.

Defaulting to ```MPI_COMM_WORLD``` would actually be a very bad idea: The implicit parallel constructor would then be equivalent to the explicit serial constructor.

Reply:
I agree to both, still a (to be determined) default for `AccessType` would be (if any) the only one we might want to put a default on so we could keep it last.